openmetadata-service/src/main/java/org/openmetadata/service/resources/dqtests/TestCaseResource.java

```diff
@@ -378,7 +389,10 @@ public ResultList<TestCase> listFromSearch(
     searchListFilter.addQueryParam("testPlatforms", testPlatforms);
     searchListFilter.addQueryParam("q", q);
     searchListFilter.addQueryParam("excludeFields", SEARCH_FIELDS_EXCLUDE);
+    searchListFilter.addQueryParam("includeFields", includeFields);
     searchListFilter.addQueryParam("domain", domain);
+    searchListFilter.addQueryParam("tags", tags);
```

Comment:
do we want to expose tags field vs tier. I do see tags might be helpful, but tier should be separate filter.

Reply:
We can, it will be the same but can be better semantically for users?